packages.cake

```diff
@@ -111,6 +111,7 @@ public class Packages
         var SplatVersion = packageVersions["Splat"].FirstOrDefault().Item1;
         var SpracheVersion = packageVersions["Sprache"].FirstOrDefault().Item1;
         var SystemReactiveVersion = packageVersions["System.Reactive"].FirstOrDefault().Item1;
+        var ReactiveUIVersion = packageVersions["reactiveui"].FirstOrDefault().Item1;
```

Comment:
I've started some kind of refactoring for dependency version detection, see
https://github.com/jkoritzinsky/Avalonia/blob/a0cfa2608b22c437f69563c1d09d0e333dcd4b8b/packages.cake#L250 and https://github.com/jkoritzinsky/Avalonia/blob/a0cfa2608b22c437f69563c1d09d0e333dcd4b8b/packages.cake#L37

We probably want to remove `Dep` overload from there and add some logic for own packages.